Sources/JWTKit/JWTSerializer.swift

```diff
@@ -9,7 +9,7 @@ struct JWTSerializer {
         cty: String? = nil,
         jsonEncoder: any JWTJSONEncoder
     ) throws -> String
-        where Payload: JWTPayload
+        where Payload: Encodable
```

Comment:
Similarly, rather than duplicate this internal method, I loosened the protocol requirements to it works for both.